targets/targets.json

```diff
@@ -2598,7 +2607,8 @@
             "TRNG",
             "FLASH",
             "QSPI",
-            "MPU"
+            "MPU",
+            "WATCHDOG"
```

Comment:
This is not needed as target inherits from FAMILY_STM32 where WATCHDOG is already defined